modules/gapi/test/common/gapi_video_tests_common.hpp

```diff
@@ -43,7 +43,7 @@ inline void initTrackingPointsArray(std::vector<cv::Point2f>& points, int width,
 
     points.clear();
     GAPI_Assert((nPointsX >= 0) && (nPointsY) >= 0);
-    points.reserve(static_cast<size_t>(nPointsX * nPointsY));
+    points.reserve(static_cast<size_t>(nPointsX) * nPointsY);
```

Comment:
lets just drop the cast completely. I believe it will make the tool happy. As  here https://github.com/opencv/opencv/blob/master/modules/gapi/test/internal/gapi_int_island_tests.cpp#L72 it works OK.

Reply:
@anton-potapov Actually I thought it would have caused a warning if I hadn't cast to size_t. But, according to CI, it is OK :) Thank you!
plz see one more time